### ISSUE TITLE
unable to install MineOS

### ISSUE THREAD

**Issue by CHRIS3000521** [IgorTimofeev/MineOS#306]
when i try to install mineOS it shows me this screen
![image](https://user-images.githubusercontent.com/76781704/103357101-a357b380-4ab2-11eb-90fd-096d3ce77939.png)

**Comment by CHRIS3000521** [comment]
i don't what i have to do so please help me

**Comment by CoolCat467** [comment]
Just so you know, it's very annoying when people come here for simple errors. Sorry, but it's true.

In the first error screen, you don't have an installed EEPROM, or the EEPROM is not programmed.
In the second error screen, something is missing for the computer to run the install script and it halted execution.

How to fix:
First, ensure your computer is built properly. Have a computer with at the very least the following:
Internet Card
OpenOS floppy disk
OpenOS EEPROM
Tier 3 HDD
Tier 3 GPU 
2x Tier 3 RAM 
Tier 3 CPU (doesn't really matter, just speeds it up significantly)

If you are not using a tier 3 computer case you will also need a floppy drive.

Insert the OpenOS floppy disk to computer, turn the computer on, and once OpenOS boots type the following to console to write the operating system to the installed hard drive. **Note, capitalization is very important for pastebin to select the correct file.**
`pastebin run 0nM5b1jU`
You can paste it into console using middle mouse button or insert key (by default, changeable in Minecraft's controls settings.)

After you run the pastebin installation command, MineOS will boot it's installer script and will install onto your computer's hard drive after you fill out installation settings.

In some cases, the multiplayer server you are playing on or your single player game's internal server may block pastebin from retrieving installation script. In the event that this happens, try running the following github installation script instead:
`wget -f https://raw.githubusercontent.com/IgorTimofeev/MineOS/master/Installer/BIOS.lua /tmp/bios.lua && flash -q /tmp/bios.lua && reboot`
Again, you can paste it into console using middle mouse button or insert key (by default, changeable in Minecraft's controls settings.)

If this fails as well, then too bad. If anything, you could try to manually copy the files from github into your savegame's opencomputers folder at your computer's hard drive address, but this is complicated, it might not work, and I'm not explaining it.

**Comment by CoolCat467** [comment]
Again in French (and probably bad French too because it's just google translate)

Juste pour que vous le sachiez, c'est très ennuyeux quand les gens viennent ici pour de simples erreurs. Désolé, mais c'est vrai.

Dans le premier écran d'erreur, vous n'avez pas d'EEPROM installée ou l'EEPROM n'est pas programmée.
Dans le deuxième écran d'erreur, il manque quelque chose à l'ordinateur pour exécuter le script d'installation et il a interrompu l'exécution.

Comment réparer:
Tout d'abord, assurez-vous que votre ordinateur est correctement construit. Avoir un ordinateur avec au moins les éléments suivants:
Carte Internet
Disquette OpenOS
EEPROM OpenOS
Disque dur de niveau 3
GPU de niveau 3
2x RAM de niveau 3
Processeur de niveau 3 (cela n'a pas vraiment d'importance, il l'accélère simplement de manière significative)

Si vous n'utilisez pas de boîtier d'ordinateur de niveau 3, vous aurez également besoin d'un lecteur de disquette.

Insérez la disquette OpenOS dans l'ordinateur, allumez l'ordinateur et, une fois qu'OpenOS démarre, tapez ce qui suit dans la console pour écrire le système d'exploitation sur le disque dur installé. ** Notez que la mise en majuscules est très importante pour pastebin pour sélectionner le bon fichier. **
`pastebin run 0nM5b1jU`
Vous pouvez le coller dans la console à l'aide du bouton central de la souris ou de la touche d'insertion (par défaut, modifiable dans les paramètres de contrôle de Minecraft.)

Après avoir exécuté la commande d'installation pastebin, MineOS démarrera son script d'installation et s'installera sur le disque dur de votre ordinateur après avoir rempli les paramètres d'installation.

Dans certains cas, le serveur multijoueur sur lequel vous jouez ou le serveur interne de votre jeu solo peut empêcher pastebin de récupérer le script d'installation. Dans le cas où cela se produit, essayez d'exécuter le script d'installation github suivant à la place:
`wget -f https://raw.githubusercontent.com/IgorTimofeev/MineOS/master/Installer/BIOS.lua /tmp/bios.lua && flash -q /tmp/bios.lua && reboot`
Encore une fois, vous pouvez le coller dans la console en utilisant le bouton central de la souris ou insérer la clé (par défaut, modifiable dans les paramètres de contrôle de Minecraft.)

Si cela échoue également, tant pis. Si quoi que ce soit, vous pouvez essayer de copier manuellement les fichiers de github dans le dossier opencomputers de votre sauvegarde à l'adresse du disque dur de votre ordinateur, mais c'est compliqué, cela peut ne pas fonctionner, et je ne l'explique pas.

**Comment by CHRIS3000521** [comment]
I understand that this is annoying but these are the messages that appear after using your commands so it's not my fault

**Comment by CHRIS3000521** [comment]
but creating an extension of the computercraft mod with a floppy disk that directly contains mineOs would still be the best choice to avoid having to send you our problems unfortunately I'm not a developer so it's just a suggestion

**Comment by CoolCat467** [comment]
No, that makes people install a whole different mod, even though I've been told it's not too hard to make things compatable for OpenComputers.

MineOS never has and likely never will support ComputerCraft.

**Comment by CHRIS3000521** [comment]
allright but a think the guys like me prefer install a another mod rather what to have an error

**Comment by CHRIS3000521** [comment]
and me i'm use OpenComputers 1.12.2

**Comment by CoolCat467** [comment]
Did previous help your installation problem? I've seen from other comments that somehow removing and replacing the screen somehow can fix problems too.

**Comment by CHRIS3000521** [comment]
no it's always the even problem

**Comment by CoolCat467** [comment]
?

**Comment by CHRIS3000521** [comment]
if i enter command the even screen is show

**Comment by CoolCat467** [comment]
Huh

**Comment by CoolCat467** [comment]
What is the "even screen"?

**Comment by CHRIS3000521** [comment]
it's still the screen I showed you at the start

**Comment by CHRIS3000521** [comment]
it's always a blue screen

**Comment by CoolCat467** [comment]
Then install a lua EEPROM by crafting an EEPROM with the OpenComputers manual and shift click that item into the right slot on your computer.

**Comment by CoolCat467** [comment]
No one is writing a mod just for a boot floppy when there's github or pastebin to do effectively do that already.

**Comment by CHRIS3000521** [comment]
here what i do https://www.youtube.com/watch?v=IXOCGyxThEw

**Comment by CoolCat467** [comment]
Interesting... Since neither commands work, your internet service provider or any firewalls/antivirus software on your computer may be preventing the installer from downloading. I know for a fact the installer works, because just yesterday I used in in-game myself. That is very strange that neither command worked...

**Comment by CHRIS3000521** [comment]
so you tell me to try disabling my antivirus before trying again

**Comment by CHRIS3000521** [comment]
but just what is the open computers version advise to use MineOS

**Comment by CoolCat467** [comment]
OpenComputers is available in many different versions of Minecraft, so I would assume the most current OpenComputers version.

**Comment by CHRIS3000521** [comment]
thank you it's work now thank you at 100%

**Comment by IgorTimofeev** [comment]
![image](https://user-images.githubusercontent.com/11760002/104768052-9cfa5480-579f-11eb-98eb-abc038e11d6c.png)
